sematic/api/endpoints/resolutions.py

```diff
@@ -193,11 +198,22 @@ def schedule_resolution_endpoint(
         if "rerun_from" in flask.request.json:
             rerun_from = flask.request.json["rerun_from"]
 
-    resolution = schedule_resolution(
-        resolution=resolution, max_parallelism=max_parallelism, rerun_from=rerun_from
+    jobs = get_jobs_by_run_id(resolution_id, kind=JobKind.resolver)
+    resolution, post_schedule_jobs = schedule_resolution(
+        resolution=resolution,
+        max_parallelism=max_parallelism,
+        rerun_from=rerun_from,
+        existing_jobs=jobs,
+    )
+    logger.info(
+        "Scheduled resolution with job %s/%s",
+        post_schedule_jobs[-1].namespace,
+        post_schedule_jobs[-1].name,
     )
 
     save_resolution(resolution)
+    for job in post_schedule_jobs:
+        save_job(job)
```

Comment:
In reality there would only ever be one job in this list. I admit that the way it is structured is kind of confusing (it was meant to mirror run scheduling, which can have multiple jobs); I've now refactored so that it is clear there is always just one job here.